lib/takeaway.rb

```diff
@@ -0,0 +1,33 @@
+class Restaurant
+    attr_reader :menu, :order
+
+    def initialize
+        @menu = { 'burger': 5,
+                  'pizza': 6,
+                  'lasagna': 4,
+                  'ice_cream': 2
+                }
+        @order = []
```

Comment:
Are you going to create a new restaurant for each order? I suggest a new class is needed for the customer.